src/metric_model_reporter.h

```diff
@@ -79,7 +79,9 @@ struct MetricReporterConfig {
   bool is_decoupled_ = false;
 
  private:
-  // Maps the metric fullname to its lookup key.
+  // Maps the metric family fullname to its lookup key. This field is required
+  // to find the lookup key given its fullname when users specify in the custom
+  // metric configuration "ModelMetrics".
```

Comment:
```suggestion
  // Maps the metric family fullname to its lookup key. This field is required
  // because the users are expected to configure metric configuration "ModelMetrics"
  // with the full name displayed from metrics reporting while a different name is used internally .
```